config/initializers/rack_attack.rb

```diff
@@ -21,9 +19,7 @@ class Rack::Attack
 
```

Comment:
(Continuing the same comment chain - we could also, identically to what we're doing here with `url = ...`, just define a regular `safe_ips` variable inside the `class Rack::Attack` definition from the `ENV.fetch` value)